src/components/AppFrame.js

```diff
@@ -270,7 +270,16 @@ class AppFrame extends React.Component {
         }
 
         const responder = (response) => this.Respond(response.requestId, source, response);
-        await this.props.rootStore.client.CallFromFrameMessage(event.data, responder);
+
+        if(
+          event.data.args &&
```

Comment:
may want to add validation to throw on a supplied but unrecognized value (saves everyone time if a dev fat-fingers)